packages/sdk/react-client/src/util/useObservable.ts

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import { useEffect, useState } from 'react';
+
+import type { MulticastObservable } from '@dxos/async';
+
+export const useObservable = <T = any>(observable: MulticastObservable<T>) => {
```

Comment:
Great minds think alike, @dmaretskyi created a hook for this already https://github.com/dxos/dxos/blob/main/packages/common/react-async/src/useMulticastObservable.ts

Reply:
Ah, duh, thank you!